cmake/developer_package/compile_flags/os_flags.cmake

```diff
@@ -382,9 +382,12 @@ if(CMAKE_CXX_COMPILER_ID STREQUAL "MSVC")
     ov_add_compiler_flags(/wd4275)
 
     # Enable __FILE__ trim, use path with forward and backward slash as directory separator
-    add_compile_options(
-        "$<$<COMPILE_LANGUAGE:CXX>:/d1trimfile:${OV_NATIVE_PROJECT_ROOT_DIR}\\>"
-        "$<$<COMPILE_LANGUAGE:CXX>:/d1trimfile:${OpenVINO_SOURCE_DIR}/>")
+    # github actions use sccache which doesn't support /d1trimfile compile option
+    if(NOT DEFINED ENV{GITHUB_ACTIONS})
```

Comment:
Is it safe to use different compile options in CI? I mean, could it lead to some issues that won't be reproduced locally due to this condition (or vise versa)?